vscode-dotnet-runtime-extension/src/extension.ts

```diff
@@ -336,6 +338,52 @@ export function activate(vsCodeContext: vscode.ExtensionContext, extensionContex
         return pathResult;
     });
 
+    /**
+     * @returns 0 on success. Error string if not.
+     */
+    const dotnetUninstallRegistration = vscode.commands.registerCommand(`${commandPrefix}.${commandKeys.uninstall}`, async (commandContext: IDotnetAcquireContext | undefined) =>
+    {
+        return uninstall(commandContext);
+    });
+
+    async function uninstall(commandContext: IDotnetAcquireContext | undefined) : Promise<string>

```

Comment:
nit: potentially moving all command processor to separate file may improve readability of extension.

Reply:
Thanks for your review :) I think this is a good suggestion but probably worth it's own PR, so I won't do this here.